test/cli/cli.test.js

```diff
@@ -2,29 +2,34 @@ const child_process = require("child_process");
 const path = require("path");
 const pify = require("pify");
 
-const execFile = pify(child_process.execFile);
+const exec = pify(child_process.exec);
 
 describe("cli", () => {
     it("accepts a root directory", () => {
         let rootDir = path.join(__dirname, "resources");
 
-        return execFile(
+        let command = [
```

Comment:
exec takes a single string with the full command, so I abstracted the command out into its own line so it's more readable